dll/win32/shell32/shlexec.cpp

```diff
@@ -2710,6 +2710,10 @@ HRESULT WINAPI ShellExecCmdLine(
         {
             StringCchCopyW(szFile, _countof(szFile), szFile2);
         }
+        else if (SearchPathW(NULL, szFile, NULL, _countof(szFile2), szFile2, NULL))
```

Comment:
It looks like you could put this with the previous if condition: schematically speaking,
```c
        if ((PathIsRelativeW(szFile) &&
             GetFullPathNameW(szFile, _countof(szFile2), szFile2, NULL) &&
             PathFileExistsW(szFile2)) ||
            SearchPathW(NULL, szFile, NULL, _countof(szFile2), szFile2, NULL))
        { do_the_copy(); }
```

Reply:
Fixed.